docs/content/Contribution guidelines/operations/packaging-and-versioning.md

```diff
@@ -79,7 +79,7 @@ Following are some recent tags and what they correspond to.
 ## KubeStellar core Helm chart
 
 This Helm chart describes a deployment of the KubeStellar core
-components (kcp server, KubeStellar controllers, initialization) as
+components (a space provider server, KubeStellar controllers, initialization) as
```

Comment:
I would remove the word "server" here: "space provider" logically covers all of its machinery, including a server if there is one.